RadixWallet/Features/SettingsFeature/AccountSecurity/ManualAccountRecoveryScan/ManualAccountRecoveryCoordinator+Reducer.swift

```diff
@@ -7,6 +7,11 @@ public struct ManualAccountRecoveryCoordinator: Sendable, FeatureReducer {
 
 	public struct State: Sendable, Hashable {
 		public var path: StackState<Path.State> = .init()
+		public var isMainnet: Bool
```

Comment:
Very minor - I guess we should/could be consistent in how we initialise properties - inline or in the initialiser.

Reply:
sure